operator/cmd/mesh/install.go

```diff
@@ -182,6 +189,17 @@ func runApplyCmd(cmd *cobra.Command, rootArgs *rootArgs, iArgs *installArgs, log
 		return fmt.Errorf("failed to install manifests: %v", err)
 	}
 
+	// Workaround for broken validation with revisions (https://github.com/istio/istio/issues/28880)
+	// TODO(Monkeyanator) remove once we have a nicer solution
+	if iArgs.revision != "" {
+		if !requiresIstiodServiceCreation(clientset, name.IstioDefaultNamespace) {
```

Comment:
would this be `if requiresIstiodServiceCreation(clientset, name.IstioDefaultNamespace) {` instead?

Reply:
Yes, thanks for catching